src/tests/mocks/index.ts

```diff
@@ -13,6 +13,7 @@ export const mockCompoEntry: ICompoEntry = {
     name: 'Stochastic Mock Coverage Tracer',
     description: '',
     creator: 'anon',
+    platform: 'PC',
     entryfile_url: 'http://intanssi.org/files/123.zip',
```

Comment:
intanssi ? :P Not that it matters.

Reply:
intanssiiiii

Reply:
Okay, such a strong argument that this discussion is obviously resolved.